package.json

```diff
@@ -20,9 +20,9 @@
   },
   "scripts": {
     "build": "gatsby build",
-    "build-beta": "BUILD_ENV=development npm run build",
+    "build-beta": "cross-env BUILD_ENV=development npm run build",
```

Comment:
Likewise, we've enabled `cross-env` here. This _should_ function as-intended, but just wanted to highlight